maintenancemode/models.py

```diff
@@ -24,12 +24,12 @@ def ignored_url_patterns(self):
         qs = self.ignoredurl_set.values_list(
             "pattern", flat=True
             )
-        return map(str, qs)
+        return list(qs)

```

Comment:
This is already a list when returned from `.values_list()`. Was there a particular reason for passing `qs` to `list()` ?

Reply:
Manager methods always return querysets unless explicitly converted somewhere like here.

Reply:
Doh. That's correct. I don't work with Django very much anymore, and completely forgot.